companies/views.py

```diff
@@ -214,6 +214,18 @@ def add_team(request):
     return render(request, "companies/add_team.html", {"form": form})
 
 
+@login_required
+@is_company_owner
+def delete_team(request, name):
```

Comment:
This should take PK as the argument. This will prevent issues if the user has any strange symbols in the team name or any spaces.